.storybook/preview.tsx

```diff
@@ -29,14 +31,24 @@ export const decorators = [
   Story => (
     <SiteProvider>
       <LocaleProvider>
-        <ThemeProvider font={openSans.style.fontFamily}>
-          <div data-test-id="story-root">
+        <ThemeProvider>
+          <div
+            data-test-id="story-root"
+            className={`${openSans.variable} font-sans`}
```

Comment:
I don't think you need the "font-sans" here.

Reply:
I have tested without and for and weird reason it's not working without `font-sans` 🤔

Reply:
What is the rendered content of this className?

Reply:
with 'font-sans'
```
__variable_open-sans-normal font-sans
```
without
```
__variable_open-sans-normal
```

Reply:
> I don't think you need the "font-sans" here.

Yes, It is [necessary](https://nextjs.org/docs/pages/building-your-application/optimizing/fonts#with-tailwind-css).